docs/ref/contrib/gis/geoip2.txt

```diff
@@ -0,0 +1,194 @@
+=======================
+Geolocation with GeoIP2
+=======================
+
+.. module:: django.contrib.gis.geoip2
+   :synopsis: High-level Python interface for MaxMind's GeoIP C library.
+
+The :class:`GeoIP2` object is a wrapper for the
+`MaxMind geoip2 Python library`__. [#]_
+
+In order to perform IP-based geolocation, the :class:`GeoIP2` object requires
+the geoip2 Python library and either the GeoIP `Country` or `City` datasets in
+binary format (the CSV files will not work!).
+
+These datasets may be `downloaded from MaxMind`__.  Grab the
+``GeoLite2-Country.mmdb.gz`` and `` GeoLite2-City.mmdb.gz`` files and unzip them
+in a directory corresponding to what you set :setting:`GEOIP_PATH` with in your
+settings.  See the example and reference below for more details.
+
+Additionally, it is recommended to install the `libmaxminddb C library`__, so
+that``geoip2`` can leverage the C library's faster speed.
+
+__ http://geoip2.readthedocs.org/
+__ http://dev.maxmind.com/geoip/geoip2/geolite2/
+__ https://github.com/maxmind/libmaxminddb
+
+Example
+=======
+
+Here is an example of its
+usage::
+
+     >>> from django.contrib.gis.geoip2 import GeoIP
+     >>> g = GeoIP2()
+     >>> g.country('google.com')
+     {'country_code': 'US', 'country_name': 'United States'}
+     >>> g.city('72.14.207.99')
+     {'city': 'Mountain View',
+     'country_code': 'US',
+     'country_name': 'United States',
+     'dma_code': 807,
+     'latitude': 37.419200897216797,
+     'longitude': -122.05740356445312,
+     'postal_code': '94043',
+     'region': 'CA'}
+     >>> g.lat_lon('salon.com')
+     (37.789798736572266, -122.39420318603516)
+     >>> g.lon_lat('uh.edu')
+     (-95.415199279785156, 29.77549934387207)
+     >>> g.geos('24.124.1.80').wkt
+     'POINT (-95.2087020874023438 39.0392990112304688)'
+
+``GeoIP`` Settings
+==================
+
+.. setting:: GEOIP_PATH
```

Comment:
I guess that would be useful if someone wanted to use the classes side-by-side. Not sure that would be the case though.